blottertrax/helper/templates.py

```diff
@@ -47,3 +48,17 @@
 
 If you have any questions or believe that there has been an error, you may [PM the moderators](http://www.reddit.com/message/compose?to=/r/listentothis&subject=Please review my post). You may also [click here to see our full rule set](https://www.reddit.com/r/listentothis/wiki/rules).
 '''
+
+modmail_reply: str = '''
+/u/{},
+
+If you are contacting us about your [recent submission]({}), it was likely removed due to your account being too new to participate in our community.  Please continue to participate by leaving comments on r/listentothis and elsewhere on reddit and then come back to submit some music. As policy, we don't disclose the specific karma minimum for submitting.
+
+Furthermore, please carefully review [our rules](https://www.reddit.com/r/listentothis/wiki/rules) when you return.
+
+^^This message was sent by an automated bot.  If you believe this message does not resolve your issue please feel free to reply.
+'''
+
+
+def get_modmail_reply_new_account(username: str, recent_submission: Submission):
```

Comment:
Don't know if this method is that helpful, it's not how we have used templates before

Reply:
This might just be something I don't "get" about Python.  With such a long string interpolation I like abstracting it away and providing a function that gives very useful autocomplete names as to what params are in which order.  The calling function doesn't have to care if we reorganize the template if we keep the parameters in the template helper the same.

For instance, today if we wanted to change

```
Hey /u/{}, your post is being removed because of {}.
```

to 

```
Post removed for: {}.  Please try again /u/{}.
```

We have to change the call signature for getting that template everywhere we use it.

This function provides the ability to simply swap it within the function itself and keep the params passed in the same way such that calling code doesn't need to care.

Reply:
To clarify, I suppose I'm proposing adding these helper functions for all other templates instead of removing this one.

Thoughts?

Reply:
You make a strong case, I'm for doing this